.github/pymeasure_np2.yml

```diff
@@ -0,0 +1,29 @@
+name: pymeasure_np2
+channels:
+  - conda-forge
+dependencies:
+  - cloudpickle=1.6.0
+#  - pandas>=2.2.2  # The version on conda-forge seems not be built with numpy==2, but the pip version has been
+#  - pint=0.18  # A pint version that has not yet been released (>0.23) is required to work with numpy2
+  - pyqt=5.15.7
+  - pyqtgraph=0.12.4
+  - pyserial=3.4
+  - pyvisa=1.12.0
+  - pyzmq=24.0.1
+  - qt=5.15.6
+# Development dependencies below
+  - pytest-qt=4.2.0
+  - pytest-runner=5.2
+  - pytest=7.2.0
+  - pytest-cov=4.1.0
+  - pyvisa-sim==0.5.1
+  - flake8=6.0.0
+  - setuptools_scm # don't pin, to get newest features
+  - sphinx=5.3.0
+  - sphinx_rtd_theme=1.2.2
+#  pip is currently not needed, but the recommended tool when packages that are unavailable on conda are required
+  - pip  # don't pin, to gain newest conda compatibility fixes
+  - pip:
+      - numpy==2.0.0rc1
+      - pandas==2.2.2
+      - -e git+https://github.com/hgrecco/pint.git#egg=pint
```

Comment:
Is the "-e" for an editable install? Is it needed?

Reply:
Oh, I thought it was based on some testing and the documentation, but it might not be strictly necesary